src/box/lua/schema.lua

```diff
@@ -2509,6 +2518,23 @@ base_index_mt.count_luac = function(index, key, opts)
     return internal.count(index.space_id, index.id, itype, key);
 end
 
+-- 1-based iterator-relative index of the first matching tuple. If the such a
+-- tuple does not exist, returns the index it would be located at if existed.
```

Comment:
```suggestion
-- tuple does not exist, returns the index that would be located at if existed.
```

Reply:
Reworded with "the index at which it would be located" to make it clear. Please let me know if you're OK with this.